### DISCUSSION TITLE
Add o the curseforge page that if you play a mod pack before you report a bug check the mod pack first

### DISCUSSION THREAD

**Discussion by JaegerPhantom** [Aizistral-Studios/Enigmatic-Legacy#291] [Ideas]
People seem to think that before they ask the modpack maker if they disabled something they will approach this mod and ask if it is a bug, maybe put it in big red letters on top of the CF page that before the mod pack maker were asked there is no need to say it is a bug.

**Comment by Aizistral** [comment]
I have already done a similar disclaimer about Better Minecraft Plus once, but there isn't really a good way for me to prevent this on a more generic level. Most of the time people correctly recognize bugs, even if they are not caused by the mod itself.
